### DISCUSSION TITLE
Support Swift 5.7

### DISCUSSION THREAD

**Discussion by wadetregaskis** [SwiftPackageIndex/SwiftPackageIndex-Server#2010] [Ideas]
Currently the build bots don't support Swift 5.7.  This is particularly problematic for 5.7+ packages (e.g. SwiftWeather) since it gives the impression that they're broken outright (all builds are shown as failing, for all OSes and Swift versions).

**Comment by daveverwer** [comment]
Thanks for the feedback, Wade! We plan to support 5.7 on the build matrix shortly after the final release. I can't give you an exact time, but it will be high on our priorities.

**Reply by wadetregaskis** [reply]
I believe this can be closed as completed?

**Reply by daveverwer** [reply]
This never made it to be an issue as it was already in our plans. We don't tend to close discussions as the only option is to "lock" them and that feels overly harsh

Thanks for checking back in, though!

**Reply by RemiBardon** [reply]
We can mark a comment as the answer, as I did for https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/discussions/2442#discussioncomment-6203395. That shows the discussion is resolved as an issue would. Maybe that's what you're after. I suppose only authors can mark comments as answers though.

**Reply by Sherlouk** [reply]
Looks like it's a new feature to resolve without locking, hopefully that helps clarify things 👍  Thanks Wade for raising the discussion.
